### ISSUE TITLE
mark as watched (feature request) 

### ISSUE THREAD

**Issue by nguyenthanhvu** [YePpHa/YouTubeCenter#1739]
Ideally what I'd like to have is a little  "X" in the lower left corner of each thumbnail that appears on my homepage, so I could check off and hide the ones I'm not interested in viewing, which frees up space on my homepage for other subscriptions.

**Comment by Lukas45** [comment]
You mean like this? =)
![6959de447a7d7421f4974e5b5166ac61 1](https://cloud.githubusercontent.com/assets/4707702/7444852/aee55e8e-f197-11e4-9ccf-f05623832853.png)

**Comment by Zarggg** [comment]
No, the OP means that they want a manual way to mark a video as "watched" without actually watching it.

+1 for this request.
